### PR TITLE
Pelux sde docs

### PR DESCRIPTION
**Pull request by DunderRoffe**
Some docs on PELUX-SDE

### CODE REVIEW

File: docs/articles/sde/introduction_to_sde.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Software Development Environment
+********************************
+
+PELUX Software Development Environment (SDE) is a tool that creates a virtual
+machine  with all the tools needed when developing towards a PELUX platform.
+More specifically the SDE is setup using vagrant to generate a VirtualBox
+instance. In order to facilitate this, the PELUX SDE requires an instance of
+PELUX SDK to be present while setting up the environment. The source code can
+be found at https://github.com/Pelagicore/PELUX-SDE. Please read the README.md
```
Comment (tobsan) [inline docs/articles/sde/introduction_to_sde.rst]: Turn this link into a link instead, if you know what I mean

**Review by tobsan — CHANGES_REQUESTED** [review]

File: docs/articles/sde/introduction_to_sde.rst (hunk 1 above)
Comment (DunderRoffe) [reply docs/articles/sde/introduction_to_sde.rst]: like this?

**Comment by DunderRoffe** [comment]
> Turn this link into a link instead, if you know what I mean

@tobsan like this?

**Review by tobsan — APPROVED** [review]

File: docs/articles/sde/introduction_to_sde.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Build
+-----
+Dependencies:
+* Vagrant
+* VirtualBox
+
+.. code-block:: bash
+
+    vagrant up
```
Comment (rpannek) [inline docs/articles/sde/introduction_to_sde.rst]: Mention checking out the repo

File: docs/articles/sde/introduction_to_sde.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
+:orphan:
+
+Software Development Environment
+********************************
+
+PELUX Software Development Environment (SDE) is a tool that creates a virtual
+machine  with all the tools needed when developing towards a PELUX platform.
+More specifically the SDE is setup using vagrant to generate a VirtualBox
+instance. In order to facilitate this, the PELUX SDE requires an instance of
+PELUX SDK to be present while setting up the environment.
```
Comment (rpannek) [inline docs/articles/sde/introduction_to_sde.rst]: Where should it be present?

File: docs/articles/sde/introduction_to_sde.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Following is a list of environment variables used by vagrant when provisioning
+the SDE.
+
+* SDK_FILE_NAME: Name of the self-extracting SDK package to install into the
+SDE. Can include wildcards. Defaults to `oecore*toolchain*sh`.
+* NO_GUI: Will create a headless SDE when set.
+
+
+Testing
```
Comment (rpannek) [inline docs/articles/sde/introduction_to_sde.rst]: Hint on who would need to run that.

File: docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+.. taglist:: howto
+
+Prerequisites:
+--------------
+#. Hardware target is running PELUX
+#. Hardware target has an sftp server installed
+
+Steps
+-----
+#. Open an instance of the PELUX SDE
```
Comment (rpannek) [inline docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst]: How do I do that? Do I doubleclick on something?

File: docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+#. Start Qt Creator
+#. Open up tools->build & run->kits and verify that the PELUX SDE installed a kit successfully
+#. Under devices, create a Generic Linux Device with the IP Address and login to your hardware target
+#. Open any Qt project that you want to run on the hardware target.
+#. Under project->run settings add any --platform arguments required to run applications.
```
Comment (rpannek) [inline docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst]: Add that if you have other args you need to set them too

**Review by rpannek — CHANGES_REQUESTED** [review]

File: docs/articles/sde/introduction_to_sde.rst (hunk 2 above)
Comment (DunderRoffe) [reply docs/articles/sde/introduction_to_sde.rst]: This is too project specific I think

File: docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst (hunk 5 above)
Comment (DunderRoffe) [reply docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst]: This should be clear through the introduction

**Comment by DunderRoffe** [comment]
@rpannek rewrote the how-to and the intro :)

**Review by rpannek — APPROVED** [review]

**Review by breakreturn — COMMENTED** [review]
I don't have time to do a proper review, but I think you've gotten enough peer review with the approvals you have.

**Pull request merged**